babel.config.js

```diff
@@ -83,6 +77,12 @@ module.exports = function getBabelConfig(api) {
   return {
     assumptions: {
       noDocumentAll: true,
+      // With our case these assumptions are safe, and the
+      // resulting behavior is equivalent to spec mode.
+      setPublicClassFields: true,
+      privateFieldsAsProperties: true,
+      objectRestNoSymbols: true,
+      setSpreadProperties: true,
```

Comment:
These are taken from https://github.com/mui/material-ui/pull/37461

I'm not sure if we actually need all of them as the browsers we target support spreading and private fields natively. But including these does no harm.